moped-editor/src/queries/staff.js

```diff
@@ -25,7 +25,6 @@ export const GET_USER = gql`
       first_name
       is_coa_staff
       last_name
-      staff_uuid
```

Comment:
I missed this too! Reminds me of your dev sync share yesterday about the challenge of keeping DB schema/backend/frontend all in sync